### PR TITLE
added sticky-footer option to theme

### PR DESCRIPTION
**Pull request by rburkey87**
also moved byu-header within #wrapper on page.html.twig (to match front
page)

### CODE REVIEW

File: css/byu_default.css [hunk 1]
```diff
@@ -266,6 +300,23 @@ byu-menu {
...
 
+#wrapper.sticky-footer {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh; }
+
+.main.sticky-footer {
+  flex-grow: 1; }
+
+#footer {
```
Comment (katrialesser) [inline css/byu_default.css]: why was this needed?

File: scss/byu_default.scss [hunk 2]
```diff
@@ -378,6 +400,29 @@ byu-search {
...
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+  }
+
+  .main.sticky-footer {
+    flex-grow: 1;
+  }
+
+  //Unsetting Porto theme footer styling (leaving a light blue bar above BYU footer)
```
Comment (katrialesser) [inline scss/byu_default.scss]: Let's just remove this from porto instead. let me know if you need me to grant you access to that porto one.

File: css/byu_default.css (hunk 1 above)
Comment (rburkey87) [reply css/byu_default.css]: It doesn't make the flex changes without it - it expands the body to the full space (from what I understand). According to w3 schools "The flex-grow property specifies how much the item will grow relative to the rest of the flexible items inside the same container." Since `#wrapper` defines the full area with `display: flex`, `.main` allows the content within the wrapper to grow to the full size of the screen.

File: scss/byu_default.scss (hunk 2 above)
Comment (rburkey87) [reply scss/byu_default.scss]: It looks like it's already been commented out in the Porto theme, which means that those who already installed the byu_d8 theme will need to download an updated copy of Porto from GitHub. Once Porto is replaced with the current version, the light blue bar is gone. So I'll remove this from the byu_d8 scss file. :)
